public/docs/ts/latest/guide/change-log.jade

```diff
@@ -5,6 +5,17 @@ block includes
   The Angular documentation is a living document with continuous improvements.
   This log calls attention to recent significant changes.
 
+  ## NEW: Added `src/` folder to all examples (2017-02-02)
+  We've updated all our examples include a `src/` folder at the project root. 
+  Having this folder helps separate app code from setup and configuration files.
+  You can read about moving your existing project to this structure in 
+  [the QuickStart repo update instructions](https://github.com/angular/quickstart#updating-to-a-newer-version-of-the-quickstart-repo).
+  * `app/` has been moved to `src/app/`
+  * `src/app/main.ts` has been moved to `src/main.ts`
```

Comment:
Shouldn't be better to say:

`app/main.ts` is not at `src/main.ts` or something like that?